--- libicu/ndk_headers/unicode/utypes.h	2024-12-10 13:51:23.539396380 +0000
+++ libicu/ndk_headers/unicode/utypes.h	2024-12-10 13:51:23.791401428 +0000
@@ -56,23 +56,6 @@
  * integer and other types.
  */
 
-/** @{ API visibility control */
-
-/**
- * \def U_SHOW_CPLUSPLUS_API
- * When defined to 1 (=default) and compiled with a C++ compiler, both C and C++ APIs are visible.
- * Otherwise, only C APIs are visible; this is for C++ users who want to
- * restrict their usage to binary stable C APIs exported by ICU DLLs.
- * \xrefitem internal "Internal"  "Internal List"  Do not use. This API is for internal use only.
- */
-/**
- * \def U_SHOW_CPLUSPLUS_HEADER_API
- * When defined to 1 (=default) and compiled with a C++ compiler, C++ header-only APIs are visible.
- * This is for C++ users who restrict their usage to binary stable C APIs exported by ICU DLLs
- * (U_SHOW_CPLUSPLUS_API=0)
- * but who still want to use C++ header-only APIs which do not rely on ICU DLL exports.
- * \xrefitem internal "Internal"  "Internal List"  Do not use. This API is for internal use only.
- */
 #ifdef __cplusplus
 #   ifndef U_SHOW_CPLUSPLUS_API
 #       define U_SHOW_CPLUSPLUS_API 1
@@ -104,7 +87,6 @@
 #define U_HIDE_INTERNAL_API 1
 #endif
 
-/** @} */
 
 /*===========================================================================*/
 /* ICUDATA naming scheme                                                     */
